src/sgame/sg_admin.cpp

```diff
@@ -1184,7 +1184,7 @@ static bool admin_match( void *admin, const void *match )
 	}
 
 	G_SanitiseString( ( ( g_admin_admin_t * ) admin )->name, n1, sizeof( n1 ) );
-	return strstr( n1, n2 ) ? true : false;
+	return !!strstr( n1, n2 );
```

Comment:
reminder from the [old thread](https://github.com/Unvanquished/Unvanquished/pull/965/files#r84266517):

@mbasaglia on 20 Oct 2016 Member
> TBH I have some mixed feelings about bang bang

@Kangz 13 days ago Member
> Same here.